### PR TITLE
feat: inspect variable value

### PR DESCRIPTION
**Pull request by TrySound**
Here added "Eye" button to inspect variable values in big editor. Useful to preview resource and parameter variables which are not specified by user.

Screenshot 2024-04-16 at 18 50 41

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Review by kof — CHANGES_REQUESTED** [review]
discussed: use the menu in the list, move both delete and inspect to the menu, later we can add copy (or right away if not hard), cut, disable/hide, duplicate,

**Comment by TrySound** [comment]
Seems like its not accessible to open dialog with menu item caz menu will close on select and dialog will not have element to return focus.

**Comment by TrySound** [comment]
image

**Review by kof — APPROVED** [review]

File: apps/builder/app/builder/shared/expression-editor.tsx [hunk 1]
```diff
@@ -433,3 +433,43 @@ export const ExpressionEditor = ({
...
+// by spliting into separate component which is invoked
+// only when dialog content is rendered
+const ValuePreviewEditor = ({ value }: { value: unknown }) => {
+  const extensions = useMemo(() => [javascript({})], []);
+  return (
+    <BaseCodeEditor
+      readOnly={true}
+      extensions={extensions}
+      value={JSON.stringify(value, null, 2)}
+      onChange={() => {}}
```
Comment (kof) [inline apps/builder/app/builder/shared/expression-editor.tsx:447]: shouldn't this be optional then?

**Pull request merged**